test/unit/services/PushClientNotificationServiceSpec.scala

```diff
@@ -37,24 +38,41 @@ class PushClientNotificationServiceSpec extends UnitSpec with MockitoSugar with
   private val mockPushNotificationServiceConnector = mock[PushNotificationServiceConnector]
   private val mockGAConnector = mock[GoogleAnalyticsSenderConnector]
   private val mockNotificationLogger = mock[NotificationLogger]
```

Comment:
`private val mockCustomsNotificationsMetricsConnector = mock[CustomsNotificationMetricsConnector]` is not used

Reply:
This is used as interactions with this are verified in the tests